charts/celeborn/values.yaml

```diff
@@ -76,10 +87,11 @@ volumes:
       diskType: SSD
       capacity: 100Gi
 
-# celeborn configurations
+# -- Celeborn configurations
 celeborn:
   celeborn.master.ha.enabled: true
   celeborn.metrics.enabled: true
+  celeborn.metrics.prometheus.path: /metrics/prometheus
```

Comment:
a bug fix or something else?

Reply:
> a bug fix or something else?

The pod monitor's metric endpoint should respect `celeborn.metrics.prometheus.path`, and I will update in later PRs.

Reply:
we should support the `celeborn.metrics.prometheus.path` absent case too.